### DISCUSSION TITLE
Remove from collection or read list in series/book page

### DISCUSSION THREAD

**Discussion by arshesney** [gotson/komga#1518] [Ideas]
### Describe your suggested feature

At the moment the only way to remve an item from a collection is by entering collection edit mode, in case of a large number of items that is quite incovenient as edit mode doesn't support filtering.

Since collections an read lists are displayed for a series or book page it would be nice to have the option to remove such item from its page by adding a "Remove" link, for example, as shown below:

![image](https://github.com/gotson/komga/assets/6453590/024aa1e0-a7da-46ff-a490-b08c5c0dd02c)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Added for read lists and collections:
<img width="403" alt="image" src="https://github.com/user-attachments/assets/153d02ed-8e15-4b7c-8bda-9762503ad8f0" />

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
